Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/modules/keysubmission/AnalyticsKeySubmissionStorage.kt

```diff
@@ -73,6 +73,11 @@ class AnalyticsKeySubmissionStorage @Inject constructor(
         defaultValue = -1
     )
 
+    val daysSinceMostRecentDateAtRiskLevelAtTestRegistration = prefs.createFlowPreference(
```

Comment:
if you only ever access it via `.value` does this need to be a flow preference or would a get/set implementation work as well?

Reply:
Yes, that's true. This was copy/paste convenience. Would still keep now as it does not hurt.